Makefile

```diff
@@ -0,0 +1,31 @@
+LSOURCES= elooplib/node.cpp elooplib/server.cpp elooplib/client.cpp elooplib/eventloop.cpp
+SOURCES = bvnServer/main.cpp
+HEADERS = ./elooplib
+
+CXX = g++
+CXXFLAGS = -std=c++17 -g -Wall -Wextra #-Werror
+
+all: bvnserver clean
+
+bvnserver: main.o eventloop.a
+	$(CXX) $(CXXFLAGS) main.o eventloop.a -o bvnserver
+
+main.o: $(SOURCES)
+	$(CXX)  -c $(CXXFLAGS) -I./elooplib $(SOURCES) 
+
+eventloop.a: eventloop.o client.o server.o node.o
+	ar rcs eventloop.a eventloop.o client.o server.o node.o
+	
+eventloop.o: 
+	$(CXX) -c $(CXXFLAGS)  $(LSOURCES)
+	
+client.o: 
```

Comment:
Makefile support target masks using percent sign: %

This way you won't need to add each cpp file